src/clj/harja/palvelin/integraatiot/vayla_rest/sahkoposti.clj

```diff
@@ -126,6 +128,10 @@
                         itmf (get-in asetukset [:tloik :toimenpideviestijono]))
         viesti-id (:viesti-id kutsun-data)
         kasitelty-vastaus (tloik-sahkoposti/vastaanota-sahkopostikuittaus jms-lahettaja db kutsun-data)
+        otsikko (:otsikko kutsun-data)
+        ;; TODO: tämän toimivuutta ei ole vielä testattu
```

Comment:
Pitääkö tästä olla huolissaan?

Reply:
Kyllä siitä vähäsen. Jäi nimittäin testaamatta. Testailen vielä tämän. Kiitos bongauksesta.

Reply:
Sain lopulta testattua ja kyllähän se toimii.